app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -86,11 +86,11 @@ class WeightedQueryStrategy
         daily_decay_factor: {
           clause: "(current_date - articles.published_at::date)",
           cases: [
-            [0, 1], [1, 0.99], [2, 0.985],
-            [3, 0.98], [4, 0.975], [5, 0.97],
-            [6, 0.965], [7, 0.96], [8, 0.955],
-            [9, 0.95], [10, 0.945], [11, 0.94],
-            [12, 0.935], [13, 0.93], [14, 0.925]
+            [0, 1], [1, 0.975], [2, 0.965],
+            [3, 0.955], [4, 0.945], [5, 0.935],
+            [6, 0.925], [7, 0.922], [8, 0.0919],
+            [9, 0.916], [10, 0.913], [11, 0.91],
+            [12, 0.907], [13, 0.904], [14, 0.901]
```

Comment:
There were three experiments running, and because of the shape of the class, were somewhat hidden.  The winning experiment looks much more like what we have in place with this PR.